Sources/starlight-swift/TwoPhaseSet.swift

```diff
@@ -0,0 +1,46 @@
+open class TwoPhaseSet<T> where T: Hashable {
+
+    var data: GSet<T>
```

Comment:
We have the dataset getter, why is this public?

Reply:
Let's make it private